docs/concepts.md

```diff
@@ -24,54 +24,40 @@ For output, being incremental means that Feldera outputs query results
 as sets of changes from the previous output. This improves performance
 and reduces output size.
 
-## Programs
+## Pipeline
 
-In Feldera, a SQL **program** is a collection of SQL DDL (table and
-view definitions):
+A Feldera **pipeline** pulls data from sources, runs it through a
+computation, and pushes the results to sinks.  The user can start,
+stop, manage, and monitor pipelines.
 
-* SQL table definitions with `CREATE TABLE` specify the format of
-  data.  Feldera does not store tables, since it is not a database, so
-  table definitions do not reserve disk space or other storage.
+[comment]: The GIFs below were generated with the utility in demo/docs-demo.
```

Comment:
This comment shows up in the compiled HTML

Reply:
Oops.

I changed it to an HTML style comment, which seems to be the way to do this with Docusaurus.